modules/i18n.py

```diff
@@ -55,27 +55,55 @@ def subscribe_keywords(language):
 def six_week_reminder_seven_days(language):
     if language == "English":
         return "{name} has their scheduled vaccination in 7 days. Without this vaccination your child will be vulnerable to deadly diseases."
-    # TODO: add other languages
+    elif language == "Hindi":
+        return '7 \xe0\xa4\xa6\xe0\xa4\xbf\xe0\xa4\xa8\xe0\xa5\x8b\xe0\xa4\x82 \xe0\xa4\xae\xe0\xa5\x87\xe0\xa4\x82 {name} \xe0\xa4\x95\xe0\xa4\xbe \xe0\xa4\x9f\xe0\xa5\x80\xe0\xa4\x95\xe0\xa4\xbe\xe0\xa4\x95\xe0\xa4\xb0\xe0\xa4\xa3 \xe0\xa4\x95\xe0\xa4\xb0\xe0\xa4\xb5\xe0\xa4\xbe\xe0\xa4\x8f\xe0\xa4\x81 \xe0\xa4\x8f\xe0\xa4\xb5\xe0\xa4\x82 \xe0\xa4\x96\xe0\xa4\xbc\xe0\xa4\xa4\xe0\xa4\xb0\xe0\xa4\xa8\xe0\xa4\xbe\xe0\xa4\x95 \xe0\xa4\xac\xe0\xa5\x80\xe0\xa4\xae\xe0\xa4\xbe\xe0\xa4\xb0\xe0\xa4\xbf\xe0\xa4\x93\xe0\xa4\x82 \xe0\xa4\xb8\xe0\xa5\x87 \xe0\xa4\xac\xe0\xa4\x9a\xe0\xa4\xbe\xe0\xa4\x8f\xe0\xa4\x81.'
+    elif language == "Gujarati":
+        return '{name} \xe0\xaa\xa8 \xe0\xab\x81\xe0\xaa\x82\xe0\xab\xad \xe0\xaa\xa6\xe0\xaa\xbf\xe0\xaa\xb5\xe0\xaa\xb8 \xe0\xaa\xae \xe0\xab\x81\xe0\xaa\x82\xe0\xaa\xb0\xe0\xaa\xb8\xe0\xab\x80\xe0\xaa\x95\xe0\xaa\xb0\xe0\xaa\xa3 \xe0\xaa\xb8 \xe0\xaa\xa6\xe0\xaa\xa8\xe0\xaa\xa6\xe0\xaa\xbf\xe0\xaa\xa4 \xe0\xaa\x9b\xe0\xab\x87. \xe0\xaa\x86 \xe0\xaa\xb0\xe0\xaa\xb8\xe0\xab\x80\xe0\xaa\x95\xe0\xaa\xb0\xe0\xaa\xa3 \xe0\xaa\xb5\xe0\xaa\x97\xe0\xaa\xb0 \xe0\xaa\xa4\xe0\xaa\xae \xe0\xaa\xb0\xe0\xab\x81\xe0\xaa\x82 \xe0\xaa\xac \xe0\xaa\xb3\xe0\xaa\x95 \xe0\xaa\x9c\xe0\xab\x80\xe0\xaa\xb5\xe0\xaa\xb2\xe0\xab\x87\xe0\xaa\xa3 \xe0\xaa\xb0\xe0\xab\x8b\xe0\xaa\x97\xe0\xab\x8b \xe0\xaa\xae \xe0\xaa\x9f\xe0\xab\x87\xe0\xaa\xb8\xe0\xab\x81\xe0\xaa\x82\xe0\xaa\xb5\xe0\xab\x87\xe0\xaa\xbf\xe0\xaa\xa8\xe0\xaa\xb6\xe0\xab\x80\xe0\xaa\xb2 \xe0\xaa\xb0\xe0\xaa\xb9\xe0\xab\x87\xe0\xaa\xb6\xe0\xab\x87.'
+
 
 def six_week_reminder_one_day(language):
     if language == "English":
         return "{name} is due for their important vaccinations in tomorrow. Please do so then."
```

Comment:
There is a typo here that was in the original (I've fixed it there). It should read: 

"{name} is due for their important vaccinations tomorrow. Please do so then.